src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly

```diff
@@ -42,6 +42,36 @@ THE SOFTWARE.
                 <!-- Similar to f:dropdownDescriptorSelector, but adds fallback content to block, and JENKINS-25130 adds per-selection help: -->
                 <j:set var="item" value="${it.getItem(request)}"/>
                 <d:taglib uri="local">
+                    <!-- this tag wraps the dropdown list in a div but only in div-based forms
+                        using a div in table-based forms will crash the layout generation -->
+                    <d:tag name="snippetizerOptionsWrapper">
+                       <j:choose>
+                           <j:when test="${divBasedFormLayout}">
+                               <style>
+                                   .snippetizer-options-wrapper .setting-input.dropdownList {
+                                       max-width: calc(100% - 2rem);
+                                   }
+
+                                   .snippetizer-help {
+                                       position: relative;
+                                   }
+
+                                   .snippetizer-help a.help-button {
+                                       position: absolute;
+                                       top: -2rem;
+                                   }
```

Comment:
I understand you'd rather not deal with a css file but inlining these feels too hacky.  The styling should be present regardless of div based layout or not.